### PR TITLE
Flutter for iOS — Plugins section

### PR DESCRIPTION
**Pull request by rock3r**
This PR is for the ninth section of the work to create the `flutter-for-ios.md` document. Writing by @niamh-power on Novoda's behalf, I am just a proxy.

Following the same rough format as the `flutter-for-android.md` document, with iOS details instead.

The following sections have been created:

* How do I access the GPS sensor?
* How do I access the Camera?
* How do I Log in with Facebook?
* How do I use Firebase features?
* How do I build my own custom native integrations?

⚠️ **This requires #984 to be merged first!** ⚠️

### CODE REVIEW

File: flutter-for-ios.md [hunk 1]
```diff
@@ -27,3 +27,1759 @@ that are most relevant to your needs.
...
+A popular plugin to access the camera is [`image_picker`](https://pub.dartlang.org/packages/image_picker).
+
+## How do I Log in with Facebook?
+
+To Log in with Facebook, you can use the
+[`flutter_facebook_login`](https://pub.dartlang.org/packages/flutter_facebook_login) community plugin.
+
+## How do I use Firebase features?
+
+Most Firebase functions are covered by [first or third party plugins](https://pub.dartlang.org/flutter/packages?q=firebase).
```
Comment (mjohnsullivan) [inline flutter-for-ios.md]: Please reference the official 1st party plug-ins here

**Comment by rock3r** [comment]
Contents covered in #993, closing this. Will address comments on that one.

File: flutter-for-ios.md [hunk 2]
```diff
@@ -27,3 +27,1759 @@ that are most relevant to your needs.
...
+ * [`firebase_storage`](https://pub.dartlang.org/packages/firebase_storage) for Firebase Cloud Storage
+ * [`firebase_messaging`](https://pub.dartlang.org/packages/firebase_messaging) for Firebase Messaging (FCM)
+ * [`flutter_firebase_ui`](https://pub.dartlang.org/packages/flutter_firebase_ui) for quick Firebase Auth integrations (Facebook, Google, Twitter and email)
+
+## How do I build my own custom native integrations?
+
+If there is platform-specific functionality that Flutter or its community
+Plugins are missing, then you can build your own following [this tutorial](https://flutter.io/developing-packages/).
+
+Flutter's plugin architecture, in a nutshell, is a lot like using an Event bus in
```
[the remark this answers is not in the document]
Comment (rock3r) [reply flutter-for-ios.md]: It's mentioning a pattern, not a specific implementation. They have event buses on iOS too.

File: flutter-for-ios.md (hunk 1 above)
Comment (rock3r) [reply flutter-for-ios.md]: Done in #993

**Pull request closed without merging**